cmd/exec/main.go

```diff
@@ -609,38 +605,26 @@ Will update the helloworld workflow and set the remote workflow variable 'data.j
 	},
 }
 
-var eventCmd = &cobra.Command{
-	Use:   "event EVENT_PATH",
+var sendEventCmd = &cobra.Command{
+	Use:   "send PATH",
 	Short: "Remotely trigger direktiv event with local files",
-	Long: `Remotely trigger direktiv event with local files. This process will trigger and create the given Event when its not already existing.
```

Comment:
The way cobra generates and formats its help information is pretty good. I don't think this example really needed to be here.